shared/logutil/logutil.go

```diff
@@ -0,0 +1,83 @@
+package logutil
+
+import (
+	"os"
+	"strings"
+
+	joonix "github.com/joonix/log"
+	"github.com/sirupsen/logrus"
+	prefixed "github.com/x-cray/logrus-prefixed-formatter"
+)
+
+// WriterHook is a hook that writes logs of specified LogLevels to specified Writer
+type WriterHook struct {
+	LogLevels []logrus.Level
+}
+
+// Fire will be called when some logging function is called with current hook
+// It will format log entry to string and write it to appropriate writer
+func (hook *WriterHook) Fire(entry *logrus.Entry) error {
+	line, err := entry.String()
+	if err != nil {
+		return err
+	}
+	//simply call the file logger Println func after removing the new line char
```

Comment:
```suggestion
```

This comment doesn't add any new information. Please use comments sparingly and only when additional information is needed. 

Why do you remove the newline character?

Reply:
The newline char is serialized to file as "/n" (literal, actual newline) at the end of each log msg. It's annoying and inconvenient to my taste. I assumed that it would be better to remove them.